features/integration/c2c.feature

```diff
@@ -0,0 +1,115 @@
+@c2c
+Feature: Contract to Contract Interaction
+  Scenario:
+    * an algod v2 client
+    * a kmd client
+    * wallet information
+    * suggested transaction parameters from the algod v2 client
+    * I create a new transient account and fund it with 100000000 microalgos.
+    * I make a transaction signer for the transient account.
```

Comment:
Have we used the `*` syntax before? The quality of the cucumber implementations varies quite a bit when it comes to implementing the spec. In particular, I would keep an eye out on go plugin. Other than that, nice trick!

Reply:
Will make a note that go should watch out